ui/shared/utils/constants.js

```diff
@@ -807,6 +807,13 @@ export const getVariantMainGeneId = ({ transcripts, mainTranscriptId, selectedMa
     entry[1].some(({ transcriptId }) => transcriptId === (selectedMainTranscriptId || mainTranscriptId)),
   ) || [])[0]
 
+export const getCompoundHetsAllGeneIds = compoundHets => compoundHets.map(({ transcripts }) => Object.keys(transcripts))
+
+export const getCompoundHetsSharedGeneIds = geneIds =>
+  (geneIds.shift().filter(sameVariantGeneIds =>
```

Comment:
This is really hard to read. What format are `geneIds` in?

Reply:
`geneIds` is an array of array of strings, and the method is trying to find the first common string amongst the arrays of strings, i.e., the shared geneId amongst the compound hets.